### PR TITLE
Update to v0.14.0-rc3

### PR DESCRIPTION
**Pull request by JordanMartinez**
Backlinking to purescript/purescript#3942

### CODE REVIEW

**Comment by JordanMartinez** [comment]
CI fails with this error:
```
> eslint src && spago build --purs-args '--censor-lib --strict'


/home/runner/work/purescript-web-html/purescript-web-html/src/Web/HTML.js
Error:   1:11  error  'window' is already defined as a built-in global variable  no-redeclare

/home/runner/work/purescript-web-html/purescript-web-html/src/Web/HTML/HTMLImageElement.js
Error:   1:11  error  'Image' is already defined as a built-in global variable  no-redeclare

✖ 2 problems (2 errors, 0 warnings)
```

**Comment by kl0tl** [comment]
This happens because of the `/* global window */` and ` /* global Image */` comments in src/Web/HTML.js and src/Web/HTML/HTMLImageElement.js. We can remove them.

**Review by kl0tl — APPROVED** [review]

File: spago.dhall [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{-
+Welcome to a Spago project!
+You can edit this file as you like.
+-}
```
Comment (kl0tl) [inline spago.dhall]: Shall we remove this comment?

Comment (JordanMartinez) [reply spago.dhall]: Sure. I'm actually wondering why that comment still exists. I think I'll open an issue.

**Review by thomashoneyman — APPROVED** [review]

**Pull request closed without merging**